modules/virt-configuring-certificate-rotation.adoc

```diff
@@ -0,0 +1,42 @@
+
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/vm_networking/virt-configuring-certificate-rotation.adoc
+
+[id="virt-configuring-certificate-rotation_{context}"]
+
+Configure certificate rotation parameters to replace existing certificates. You can do this during OpenShift Virtualization installation in the web console or after installation in the YAML file.
+
+.Procedure
+
+. Open the `HyperConverged` Custom Resource (CR) by running the following command:
++
+[source,terminal]
+----
+$ oc edit hco -n openshift-cnv kubevirt-hyperconverged
+----
+
+. Edit the `spec.certConfig` fields below. All values must be 10 minutes in length or greater to avoid overloading the system, and should be expressed as strings that comply with golang's ParseDuration format (https://golang.org/pkg/time/#ParseDuration):
+
++
+[source,yaml]
+----
+apiVersion: hco.kubevirt.io/v1beta1
+kind: HyperConverged
+metadata:
+ name: kubevirt-hyperconverged
+ namespace: openshift-cnv
+spec:
+  certConfig:
+    ca:
+      duration: 48h0m0s
+      renewBefore: 24h0m0s <1>
+    server:
+      duration: 24h0m0s  <2>
+      renewBefore: 12h0m0s  <3>
+----
+<1> The value of `ca.renewBefore` should be less than or equal to the value of `ca.duration`.
+<2> The value of `server.duration` should be less than or equal to the value of `ca.duration`.
+<3> The value of `server.renewBefore` should be less than or equal to the value of `server.duration`.
```

Comment:
Change all instances of "should" to "must" (IBM style guide)

should v
Always prefer a more direct verb or way of saying something. Omit “should,” or use “must” or “might,” as appropriate.
For example, use “Reset the parameters,” not “You should reset the parameters.”